src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -313,41 +313,48 @@ def __init__(
                 )
 
         try:
-            if Path(case_filepath).match("*.cas.h5"):
-                file = h5py.File(case_filepath)
+            if Path(settings_filepath).match("*.cas.h5"):
+                file = h5py.File(settings_filepath)
```

Comment:
This code is specific to case files. We have classes to separate out the different functionality so that we don't need conditional code.